### PR TITLE
feat: add `getScreenBounds` for getting the bounds of the screen with the tray icon

### PR DESCRIPTION
**Pull request by bselwe**
- Added `getScreenBounds` method used for getting the bounds of the screen with the tray icon.

### CODE REVIEW

File: windows/tray_manager_plugin.cpp [hunk 1]
```diff
@@ -341,6 +344,46 @@ void TrayManagerPlugin::GetBounds(
...
+    result->Success();
+    return;
+  }
+
+  double devicePixelRatio =
+      std::get<double>(args.at(flutter::EncodableValue("devicePixelRatio")));
+
+  flutter::EncodableMap resultMap = flutter::EncodableMap();
+
+  HWND iconWindow = niif.hWnd;
```
Comment (bselwe) [inline windows/tray_manager_plugin.cpp:363]: @rafaelortizzableh This should essentially return the same window handle that the tray icon uses. For that window instance, we're using `MonitorFromWindow` to get the screen of the tray icon.

File: lib/src/tray_manager.dart [hunk 2]
```diff
@@ -171,6 +171,31 @@ class TrayManager {
...
   }
+
+  /// Gets the bounds of the screen where this tray icon is visible.
+  ///
+  /// TODO(bselwe): Add support for macOS.
+  ///
+  /// @platforms windows
+  Future<Rect?> getScreenBounds() async {
+    final Map<String, dynamic> arguments = {
+      'devicePixelRatio': window.devicePixelRatio,
```
Comment (rafaelortizzableh) [inline lib/src/tray_manager.dart]: We might need to replace this soon, as the [window singleton has been deprecated](https://docs.flutter.dev/release/breaking-changes/window-singleton).

Comment (rafaelortizzableh) [reply lib/src/tray_manager.dart]: @bselwe Can you change this so we use a parameter instead? That way we can eventually pass something like:
``` dart
View.of(context).devicePixelRatio
```

**Review by rafaelortizzableh — CHANGES_REQUESTED** [review]
Grat changes. Please add `devicePixelRadio` as a parameter se we can avoid issues with `window` being deprecated. Cheers!

File: lib/src/tray_manager.dart (hunk 2 above)
Comment (bselwe) [reply lib/src/tray_manager.dart]: Sure, updating!

**Pull request merged**